extensions/ipynb/src/serializers.ts

```diff
@@ -270,21 +271,17 @@ type JupyterOutput =
 
 function convertStreamOutput(output: NotebookCellOutput): JupyterOutput {
 	const outputs: string[] = [];
-	output.items
-		.filter((opit) => opit.mime === CellOutputMimeTypes.stderr || opit.mime === CellOutputMimeTypes.stdout)
-		.map((opit) => textDecoder.decode(opit.data))
-		.forEach(value => {
-			// Ensure each line is a seprate entry in an array (ending with \n).
-			const lines = value.split('\n');
-			// If the last item in `outputs` is not empty and the first item in `lines` is not empty, then concate them.
-			// As they are part of the same line.
-			if (outputs.length && lines.length && lines[0].length > 0) {
-				outputs[outputs.length - 1] = `${outputs[outputs.length - 1]}${lines.shift()!}`;
-			}
-			for (const line of lines) {
-				outputs.push(line);
-			}
-		});
+	const compressedStream = output.items.length ? compressOutputItemStreams(output.items[0].mime, output.items).toString() : '';
```

Comment:
It seems we miss a `.filter((opit) => opit.mime === CellOutputMimeTypes.stderr || opit.mime === CellOutputMimeTypes.stdout)` (which we did before)

Reply:
I removed that as I don't see how we can have an output with different mime types when dealing with stdout/stderr.
Ideally they should be different toutputs, it cannot be mixed in the same output.

Reply:
I.e. it wasn't necessary